po/fr.po

```diff
@@ -5722,43 +5732,41 @@ msgstr ""
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:4
 msgid "Display a brief help message. Identical to the help action"
-msgstr ""
+msgstr "Afficher un bref message d’aide, Identique à l’action help"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:5
-#, fuzzy
 msgid "Remove any cached packages which can no longer be downloaded"
-msgstr "Nettoyer les paquets ne pouvant plus être téléchargés"
+msgstr "Nettoyer du cache les paquets ne pouvant plus être téléchargés"
```

Comment:
Wouldn't it be even better with "supprimer du cache" instead of "nettoyer du cache"?

Reply:
Well, AFAIK, cache is always seen as a stack of garbage which we *purge* or *clean* whenever necessary, so *nettoyer* seemed to fit; you're right, though, so I'll correct.